MIPS/mip-0006-slot-reduction-90s.md

```diff
@@ -0,0 +1,270 @@
+
+Reducing the slot time will allow for more frequent block creation, improving the network's throughput and user experience.
+
+# Specification
+
+Following protocol modifications are required to implement this change:
+
+1. Update the `block_window_duration_ms` constant from 180,000 to 90,000 milliseconds
+    - Effect: Slot time will be reduced from 3 minutes to 90 seconds
+2. Update the `coinbase` constant from 720 to 360
+    - Effect: Block creators will receive half of the current MINA amount for creating a block
+    - This maintains the same token emission rate as before, ensuring inflation is not affected by the slot time change
+3. Update the account vesting schedule logic
+    - As part of the Ledger Migration procedure (that will be executed by the Mina node as part of the upgrade), update vesting parameters for **actively vesting** accounts.
+    - For the definition of **actively vesting** accounts and formulas for update of vesting parameters, see Appendix.
+4. Remove the zkapp soft limit configuration option
+    - Currently set to 24 by default
+    - Not constrained by the protocol, but will be removed completely
+
+## Prerequisites
+
+This update requires a prerequisite improvement to the SNARK work production code, which will be delivered before the hard fork as part of a soft fork update. This improvement will allow coordinators to use workers more efficiently by giving them smaller pieces for proving. See Appendix for more details.
+
+After delivering that update, the network will require at least some of the community members running the SNARK workers to run multiple (at least 4) SNARK workers per coordinator to ensure timely processing of SNARK work for the heaviest transactions.
+
+# Rationale
+
+The primary rationale for this change is to increase the frequency of block production (throughput) in the Mina blockchain while maintaining the economic model. Recent performance improvements make this change viable without sacrificing system stability. 
+
+Additionally, this change will improve latency, resulting in confirmation times for transactions cut in half and enhancing overall network responsiveness.
+
+The specific value of 90 seconds was chosen as it:
+
+1. Doubles the potential throughput of the network
+2. Remains safely within the performance capabilities of the improved system
+
+# Backwards Compatibility
+
+This MIP is not backward compatible as it changes consensus rules and requires a hard fork to implement. It is explicitly incompatible with the current "Berkeley" release and must be implemented in the next hard fork.
+
+This MIP doesn't require any changes to the protocol beyond those mentioned in the Specification section. There is no specific hard fork delivery strategy required for implementing this change.
```

Comment:
Are there delivery constraints? Will epochs reset? If not, it may be necessary to discuss timing requirements. May be helpful to clarify hard fork assumptions re: these constraints.